tests/data_files/Makefile

```diff
@@ -315,6 +315,15 @@ rsa_pkcs8_2048_public.der: rsa_pkcs8_2048_public.pem
 	$(OPENSSL) rsa -pubin -in $< -outform DER -pubout -out $@
 all_final += rsa_pkcs8_2048_public.der
 
+authorityKeyId_subjectKeyId.crt:
+	$(OPENSSL) req -x509 -nodes -days 7300 -newkey rsa:2048 -keyout authorityKeyId_subjectKeyId.crt -out authorityKeyId_subjectKeyId.crt -config authorityKeyId_subjectKeyId.conf -extensions 'v3_req'
```

Comment:
Where is the authorityKeyId_subjectKeyId.conf file?

Reply:
It was here: https://github.com/Mbed-TLS/mbedtls/pull/4161/commits/f617ad573d23eab81a6a8178657fd906b3ee3053 and then was removed https://github.com/Mbed-TLS/mbedtls/pull/4161/commits/5d5af0cede94c48ffec739a9e7b79acf59f5eb2d after the certificates were passed to test function as binary string (instead path to certtificate).

Reply:
I think that this code can be now removed, but on the other hand it would be good to have information how `authorityKeyId_subjectKeyId.crt` was generated.